checklists/avd_checklist.en.json

```diff
@@ -2,478 +2,253 @@
-            "guid": "42e52f47-21d9-428c-8b1b-d521e44a29a9",
-            "severity": "High",
-            "link": "https://azure.microsoft.com/global-infrastructure/services/?products=virtual-desktop"
+            "text": "Check Azure quotas and availability for specific VM sizes and types in the selected regions",
+            "description": "Check for specific VM SKUs, especially if you need GPU or high-specs SKUs, and eventually Azure NetApp Files availability if used.",
+            "guid": "8053d89e-89dc-47b3-9be2-a1a27f7a9e91",
+            "severity": "Low",
+            "link": "https://docs.microsoft.com/azure/azure-resource-manager/management/azure-subscription-service-limits"
         },
         {
-            "category": "Foundation",
-            "subcategory": "General",
-            "text": "Determine if hybrid connectivity is required to connect to on-premises environment",
-            "description": "If required to connect to on-premises environment, assess the current connectivity option or plan for the required connectivity. ",
-            "guid": "dd399cfd-7b28-4dc8-9555-6202bfe4563b",
-            "severity": "Medium",
-            "link": "https://docs.microsoft.com/azure/architecture/reference-architectures/hybrid-networking/"
+            "category": "Identity",
+            "subcategory": "Requirements",
+            "text": "An Azure Active Directory tenant must be available with at least one subscription linked",
+            "description": "An Azure subscription must be parented to the same Azure AD tenant, that contains a virtual network that either contains or is connected to the Windows Server Active Directory or Azure AD DS instance.",
+            "guid": "6ceb5443-5125-4922-9442-93bb628537a5",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/virtual-desktop/prerequisites?tabs=portal#identity"
         },
         {
-            "category": "Foundation",
-            "subcategory": "General",
-            "text": "Did you evaluate latency between users location and Azure AVD infrastructure?",
-            "description": "Run the 'Windows Virtual Desktop Experience Estimator' tool to select the best Azure region to deploy Host Pools. Beyond 150ms latency, user experience may be not optimal. ",
-            "guid": "0d834a34-8b26-43e6-bc60-512a26498f6d",
+            "category": "Identity",
+            "subcategory": "Requirements",
+            "text": "Review and document your identity scenario",
+            "description": "Azure Virtual Desktop supports different types of identities depending on which configuration you choose. Please review the supported scenarios mentioned in the 'More Info' article and document the design decision accordingly in the 'Comment' column. Be sure to review also the list of supported scenarios in https://learn.microsoft.com/en-us/azure/virtual-desktop/prerequisites?tabs=portal#supported-identity-scenarios.",
```

Comment:
I'm thinking that we might want to separate out the supported scenarios into its own line item, as this is easy to miss. It would force customers to check through the pre-reqs to make sure they have everything they need while then using this current link for authentication to highlight what they are proposing.

Reply:
@mikewarr if you think it would be good, feel free to submit a PR after mine. Thanks.